include/cpp2util.h

```diff
@@ -1091,13 +1099,18 @@ constexpr auto is( X const& ) -> bool {
     return true;
 }
 
+template< polymorphic C, polymorphic X >
+    requires std::same_as<C, X>
+constexpr auto is( X const& ) -> std::true_type { return {}; }
+
 template< polymorphic C, polymorphic X >
 constexpr auto is( X const& x ) -> bool {
     return Dynamic_cast<C const*>(&x) != nullptr;
 }
 
-template< polymorphic C, polymorphic X >
-constexpr auto is( X const* x ) -> bool {
+template< polymorphic C, pointer X >
+    requires polymorphic<std::remove_pointer_t<X>>
+constexpr auto is( X const& x ) -> bool {
```

Comment:
You should still take the pointer by value.

Reply:
I will check it. I had an issue with an ambiguous calls with pointers.

Reply:
OK, works! Thanks will correct it!

Reply:
Fixed!